config/locales/it.yml

```diff
@@ -105,6 +105,10 @@ it:
       searchlink: "Ricerca"
       beta-test: "Beta Test"
       invitationlink: "Richiedi un invito"
+      replieslink: "Risposte"
+      newreplieslink:
+        one: "Una risposta"
```

Comment:
`count` è un numero in cifre? Nel caso forse è più piacevole esteticamente "1 risposta".

Reply:
No, in Italiano sarebbe scorretto.